### DISCUSSION TITLE
Can't update netbox due to changed env-files

### DISCUSSION THREAD

**Discussion by fzenger7** [netbox-community/netbox-docker#1099] [Getting Help]
Hey,
I would like to update my netbox, as described at https://github.com/netbox-community/netbox-docker/wiki/Updating#how-to-update-netbox-docker.

But when I try to pull the repo, it doesn't work, because I've made changes to the env-files under env/, because I don't want to use the default passwords etc.

What am I doing wrong?
What is the intended way to set passwords and configurations for the mailserver etc?

Thanks!

**Comment by barmalej1** [comment]
It does it every time. Just backup the file in question, revert the changes using _git checkout <filename>_, run _git pull origin release_ again and follow the steps from the upgrade instructions. When the upgrade is complete restore the file from the backup and restart Netbox

**Comment by fzenger7** [comment]
Hm, okay. Is this the intended way? 
I can't find anything in the documentation about this.
Maybe it would be a solution to add the env-files to the .gitignore?

**Comment by barmalej1** [comment]
It has been like this forever. Might not be a bad idea to open a new thread in the "issues" forum
